### PR TITLE
Update design-system example.

### PR DESCRIPTION
**Pull request by anthonyshew**
Updating the design-system example!

Closes TURBO-1702

Things to pay attention to:

- [x] Is tasking correct? Are all tasks successful out-of-the-box? Do I hit cache when I'm expecting?
- [x] If I add an unused variable in a workspace, do I see a warning in both my editor and when I run turbo lint?
- [x] Does auto-importing work across packages?
- [x] Anything else wonky in your editor?
- [x] Do I get proper hot-reloading in turbo dev?
- [x] Anything else that may come up!

### CODE REVIEW

File: examples/design-system/apps/docs/stories/button.stories.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import type { Meta, StoryObj } from "@storybook/react";
-import { Button } from "ui";
+import { Button } from "@acme/ui";
```
Comment (mehulkar) [inline examples/design-system/apps/docs/stories/button.stories.tsx:2]: Did we change the org in these names to `@repo` everywhere?

File: examples/design-system/package.json [hunk 2]
```diff
@@ -1,21 +1,19 @@
 {
   "private": true,
   "scripts": {
-    "build": "turbo run build",
```
Comment (mehulkar) [inline examples/design-system/package.json:4]: We should keep `run` in these. The general advice from @nathanhammond has been to not use shorthand syntax when putting turbo commands in scripts. I generally agree.

File: examples/design-system/packages/ui/package.json [hunk 3]
```diff
@@ -12,17 +12,18 @@
   "scripts": {
     "build": "tsup src/index.tsx --format esm,cjs --dts --external react",
     "dev": "tsup src/index.tsx --format esm,cjs --watch --dts --external react",
-    "lint": "eslint src/",
+    "lint": "eslint .",
```
Comment (mehulkar) [inline examples/design-system/packages/ui/package.json:15]: In another PR (with-npm update), there was also `--max-warnings=0`. Do we want to be consistent about that?

**Review by mehulkar — APPROVED** [review]
Some fix ships on this one, nothing blocking if tests are passing

File: examples/design-system/apps/docs/stories/button.stories.tsx (hunk 1 above)
Comment (anthonyshew) [reply examples/design-system/apps/docs/stories/button.stories.tsx:2]: Went with `@acme` on this one since it's got a publishing workflow in it.

**Pull request merged**